hdf5-types/src/lib.rs

```diff
@@ -4,20 +4,57 @@
 //! Types that can be stored and retrieved from a `HDF5` dataset
 //!
 //! Crate features:
-//! * `const_generics`: Uses const generics to enable arrays [T; N] for all N.
+//! * `const-generics`: Uses const generics to enable arrays [T; N] for all N.
 //!                     Compiling without this limits arrays to certain prespecified
 //!                     sizes
+//! * `force-h5-allocator`: Force the `hdf5` allocator for varlen types and dynamic values.
```

Comment:
Can we rename this feature to `h5-allocator` or even simply `h5-alloc`? (the changelog to be updated as well in this case)

Also, should there be a matching feature in the root crate, or not?

Reply:
I'll rename this to `h5-alloc`. I think leaving this to `hdf5-sys` is alright. We havea presedence in the `static` feature not being exposed from `root`.

Reply:
👍